### PR TITLE
Jv 69 sound button placement

### LINKED ISSUE #69: Sound button placement bug
## Expected Behavior

Sound button should be displayed under the moves list inside the pokedex screen on the bottom right corner.

## Actual/Current Behavior

Sound button is being pushed down outside the main pokedex screen.

### PR DESCRIPTION
**Pull request by Wolftownn**
## Changes
1. Fixes the placement of the mute button to be on the Pokedex model itself instead of the info screen.
2. Changes the look of the mute button.

## Purpose
Gives the user a better experience when using the mute button.

## Approach
Instead of the button taking up space inside of the info screen, it now lives on the Pokedex model itself which frees up space to display more Pokemon information. 

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #69

### CODE REVIEW

**Review by connielion — APPROVED** [review]
Looks good!

File: pokemon-app/src/components/Search.js [hunk 1]
```diff
@@ -87,7 +89,7 @@ const Search = ({
                 Search Your's
               </button>
             </div>
-
+            <button className={!pokemon ? "splashScreen-mute" : "user-poke-none"}></button>
```
Comment (dannysanchez559) [inline pokemon-app/src/components/Search.js:92]: i like it

**Review by dannysanchez559 — APPROVED** [review]
"LGTM" hehe

File: pokemon-app/src/components/Search.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import React, { useState } from "react";
+import "../styles/ThemeSongs.css";
+import ThemeSongs from "./ThemeSongs";
```
Comment (Gjhnsn) [inline pokemon-app/src/components/Search.js:3]: oooooh theme songs?!

**Review by Gjhnsn — APPROVED** [review]
great work!

**Pull request merged**